Fixes
initgroups.c: In function `internal_getgrouplist':
initgroups.c:179: error: parse error before "__FUNCTION__"

CVSROOT:	/cvs/glibc
Module name:	libc
Changes by:	roland@sources.redhat.com	2001-12-16 21:52:12

Modified files:
	nss            : nsswitch.c 
	grp            : initgroups.c 

Log message:
	2001-12-16  Roland McGrath  <roland@frob.com>
	
	* nss/nsswitch.c (__nss_next): Don't use __FUNCTION__ as literal.
	* grp/initgroups.c (internal_getgrouplist): Likewise.

http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/grp/initgroups.c.diff?r1=1.26&r2=1.27&cvsroot=glibc
http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/nss/nsswitch.c.diff?r1=1.52&r2=1.53&cvsroot=glibc

Removed 2nd hunk for glibc-2.1.3 (it was just whitespace)

===================================================================
RCS file: /cvs/glibc/libc/grp/initgroups.c,v
retrieving revision 1.26
retrieving revision 1.27
diff -u -r1.26 -r1.27
--- libc/grp/initgroups.c	2001/07/06 04:54:46	1.26
+++ libc/grp/initgroups.c	2001/12/17 05:52:11	1.27
@@ -176,7 +176,7 @@
 
       /* This is really only for debugging.  */
       if (NSS_STATUS_TRYAGAIN > status || status > NSS_STATUS_RETURN)
-	__libc_fatal ("illegal status in " __FUNCTION__);
+	__libc_fatal ("illegal status in internal_getgrouplist");
 
       if (status != NSS_STATUS_SUCCESS
 	  && nss_next_action (nip, status) == NSS_ACTION_RETURN)

===================================================================
RCS file: /cvs/glibc/libc/nss/nsswitch.c,v
retrieving revision 1.52
retrieving revision 1.53
diff -u -r1.52 -r1.53
--- libc/nss/nsswitch.c	2001/07/17 08:21:36	1.52
+++ libc/nss/nsswitch.c	2001/12/17 05:52:11	1.53
@@ -178,7 +178,7 @@
     {
       /* This is really only for debugging.  */
        if (NSS_STATUS_TRYAGAIN > status || status > NSS_STATUS_RETURN)
-	 __libc_fatal ("illegal status in " __FUNCTION__);
+	 __libc_fatal ("illegal status in __nss_next");
 
        if (nss_next_action (*ni, status) == NSS_ACTION_RETURN)
 	 return 1;
